client/lib/follow-list/test/index.js

```diff
@@ -1,14 +1,12 @@
+/** @jest-environment jsdom */
+jest.mock( 'lib/wp', () => require( './lib/wp' ) );
```

Comment:
It might have lots of code inside. Otherwise I'm happy to inline it. I think I should put all mocks inside `/mocks/` subfolder to make it clearer. Sometimes I avoided it to avoid renaming files. I can revisit it now.